app/views/katello/api/v2/products/show.json.rabl

```diff
@@ -21,6 +23,31 @@ child :provider do
   attribute :name
 end
 
+attributes :sync_plan_id
+
+child :sync_plan do
+  attributes :description, :sync_date, :next_sync
+end
```

Comment:
rabl sadly doesn't merge things together here.  So you'll need to put :name and :interval here too.